lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -0,0 +1,280 @@
+package Genome::Model::ClinSeq::Command::IdentifyLoh;
+
+use strict;
+use warnings;
+use File::Spec;
+use Genome;
+
+class Genome::Model::ClinSeq::Command::IdentifyLoh {
+  is => ['Command::V2',
+        'Genome::Model::ClinSeq::Util',
+  ],
+  has_input => [
+    clinseq_build => {
+        is => 'Genome::Model::Build::ClinSeq',
+        doc => 'ClinSeq build to identify LOH regions in.
+            [Either this or a somatic variation build is required.]',
+        is_optional => 1,
+    },
+    outdir => {
+        is => 'FilesystemPath',
+        doc => 'Directory where output files will be written.',
+    },
+    somvar_build => {
+        is => 'Genome::Model::Build::SomaticVariation',
+        doc => 'SomVar build to identify LOH regions in.
+        [Either this or a clinseq build is required.]',
+        is_optional => 1,
+    },
+    minprobes=> {
+        is => 'Number',
+        is_optional => 1,
+        doc => 'Minimum number of probes to call a loh segment.',
+        default => 10,
+    },
+    segment_pc_cutoff => {
+        is => 'Number',
+        is_optional => 1,
+        doc => 'Minimum percent of LOH in a segment. n_loh_probes/n_probes * 100',
+        default => 0.95,
+    },
+    test => {
+        is => 'Boolean',
+        doc => 'set for test-cases',
+        is_optional => 1,
+        default => 0,
+    },
+    ],
+    doc => 'Identify regions of LOH using clinseq tumor/normal pairs.',
+};
+
+sub help_synopsis {
+    return <<EOS
+genome model clin-seq identify-loh\\
+    --outdir=/tmp/test/ \\
+    --clinseq-build='a4abcd1313eb4376b59e68a9dd9d5ad2'
+genome model clin-seq identify-loh\\
+    --outdir=/tmp/test/ \\
+    --somvar-build='9dc0385a1b634c9bb85eb2017b3a0c73'
+EOS
+}
+
+sub help_detail {
+    return <<EOS
+Use the results from Varscan in the somatic-variation builds and identify
+regions of LOH. Specifically, look at heterozygous single nucleotide variants
+in the normal sample and compare with stretches of homozygosity in the tumor.
+
+If a clinseq build is supplied as input, the tool attempts to use the results
+from the WGS-somatic-variation build input, if these are unavailable the tool
+looks for WEx-somatic-variation build.
+
+Alternatively, instead of a clin-seq build you can supply a
+somatic-variation build as an input.
+
+Thanks to Chris Miller for working out the workflow.
```

Comment:
I just wrote the bash script around the workflow that Nathan Dees and Dan Koboldt developed back in the day. Feel free to drop attribution here. As with many projects, this was a team effort.